### PR TITLE
fix(cyclops-ui): node card details added

### LINKED ISSUE #440: Node Conditions Update Style
The Condition cards feel out of place and need to be updated.
Screenshot 2024-07-17 at 15 14 25

To give these cards more details, add the `message`, `status`, and `lastHearbeatTime` properties. Indicate the card's status not by coloring the entire card but by giving it a check sign or danger sign in its `status` field.

Refer to [AntDesign cards](https://ant.design/components/card) for more info about changing the style of cards and adding properties.

### PR DESCRIPTION
**Pull request by s-vamshi**
closes #440

## ✅ Checks
- [ ] I have updated the documentation as required
- [x] I have performed a self-review of my code

## ℹ Additional context
added last transition time and heart beat time and message to the node conditions card
image

### CODE REVIEW

**Comment by s-vamshi** [comment]
@petar-cvit can you please review this PR?

**Comment by s-vamshi** [comment]
@petar-cvit can please review this PR in your free time?

File: cyclops-ui/src/components/pages/NodeDetails.tsx [hunk 1]
```diff
@@ -487,154 +497,67 @@ const NodeDetails = () => {
...
+                  <strong>{condition.type}</strong>
+                </h3>
+
+                <div>
+                  <strong>Last Transition Time: </strong>      
+                  <code>
+                    {new Date(
+                      condition.lastTransitionTime.toString()).toLocaleString()
+                    }
+                  </code>
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/NodeDetails.tsx:545]: Can you replace `<code>` with `<Text code>` from ant design?

**Review by petar-cvit — CHANGES_REQUESTED** [review]

**Comment by s-vamshi** [comment]
done @petar-cvit, can you please review now?

**Comment by s-vamshi** [comment]
squashed all commits

**Comment by petar-cvit** [comment]
Can you move the timestamps and the message in the next row
Screenshot 2024-07-30 at 12 02 10

**Comment by s-vamshi** [comment]
@petar-cvit made changes can you please review now?

**Review by petar-cvit — APPROVED** [review]
Thanks @s-vamshi 🧡

**Pull request merged**